rules/library.bzl

```diff
@@ -450,10 +524,58 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             tags = _MANUAL,
         )
         vendored_deps.append(import_name)
+        import_headers = native.glob(
+            ["**/*.h"],
```

Comment:
Should we get the path of `vendored_static_library` in case there are multiple libraries?

Reply:
In a static library, there is no subdirectory for a framework - so it doesn't use a path